### PR TITLE
TextGimmickHint Services

### PR DESCRIPTION
**Pull request by Small-Miao**
Add a Service to call game TextGimmick Hint
image

### CODE REVIEW

**Comment by KazWolfe** [comment]
Is there a compelling reason to make this into a new service versus adding it to `ToastGui`?

**Comment by MidoriKami** [comment]
It's objectively not a toast, and it's likely we'll want to expand it's features later.

File: Dalamud/Game/Gui/TextGimmickHint.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+using Dalamud.Plugin.Services;
+
+using FFXIVClientStructs.FFXIV.Client.UI;
+
+using static Dalamud.Plugin.Services.ITextGimmickHint;
+
+namespace Dalamud.Game.Gui
```
Comment (MidoriKami) [inline Dalamud/Game/Gui/TextGimmickHint.cs:14]: I think file scoped namespaces are preferred? Did stylecop not complain about this?

**Comment by Haselnussbomber** [comment]
CS was updated, so this is now available via `RaptureAtkModule.Instance()->ShowTextGimmickHint(...)`
No service needed.

**Pull request closed without merging**